cmd/lotus-seal-worker/main.go

```diff
@@ -259,8 +257,9 @@ var runCmd = &cli.Command{
 		}
 
 		log.Info("Opening local storage; connecting to master")
+		address := cctx.String("address")
 
-		localStore, err := stores.NewLocal(ctx, lr, nodeApi, []string{"http://" + cctx.String("address") + "/remote"})
+		localStore, err := stores.NewLocal(ctx, lr, nodeApi, []string{"http://" + address + "/remote"})
```

Comment:
This needs to be routable too, just passing 0.0.0.0 here will break cross-worker sector transfers